app/models/delivery_partnership.rb

```diff
@@ -0,0 +1,8 @@
+class DeliveryPartnership < ApplicationRecord
+  belongs_to :active_period
+  belongs_to :delivery_partner
+  has_many :school_partnerships
+
+  validates :active_period, presence: true
+  validates :delivery_partner, presence: true
```

Comment:
Hey, so just testing my understanding..

This would be a list of active periods (where a LP is a valid LP, essentially) and during these periods which delivery partners can work with them?

They can then be selected to be used for school partnerships.

This makes sense to me, but I wonder if it's easy to conflate school_partnerhips with delivery_partnerships, and it's safer to refer to a partnership as something between a school, a delivery partner and lead provider. That's why 'pairings' was originally used I think.